setup.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+from setuptools import setup, find_packages
+
+__version__ = (0, 1, 0)
+
+setup(
+    name="wps_tools",
+    description="A collection of modules used to help create wps processes",
+    keywords="wps pywps wps_tools",
+    version='.'.join(str(d) for d in __version__),
+    url="https://github.com/pacificclimate/wps-tools",
+    author="Eric Yvorchuk",
+    author_email="eyvorchuk@uvic.ca",
+    packages=find_packages(),
+    package_data={
+        'wps_tools': ["wps_tools/*.py"],
+    },
```

Comment:
I believe you are mistaking this for the `packages` portion.  This is for anything data-wise needed to run the package. Have a look [here](https://docs.python.org/3/distutils/setupscript.html#installing-package-data) for more info.